types/react-redux/tslint.json

```diff
@@ -0,0 +1 @@
+{ "extends": "dtslint/dt.json" }
```

Comment:
Initially yes, as suggested in the PR template, but I think I'll remove it due to the high amount of failures.